manifests/istio-telemetry/prometheus/templates/configmap.yaml

```diff
@@ -13,7 +13,6 @@ data:
     scrape_configs:
 
     # Mixer scrapping. Defaults to Prometheus and mixer on same namespace.
-    #
```

Comment:
I changed the scraping config to match what is used in stable/prometheus. This is largely the same as what we had before with the removal of the secure scraping job and all of the Istio specific stuff.